packages/cli/src/lib/commands/assert/options.ts

```diff
@@ -0,0 +1,27 @@
+import { SharedFlagsSettings } from 'lighthouse/types/lhr/settings';
+import { Options } from 'yargs';
+
+const budgets = {
+  alias: 'j',
+  type: 'array',
+  string: true,
+  description: 'Performance budgets (RC file only)'
+} satisfies Options;
+
+const budgetPath = {
+  alias: 'k',
+  type: 'string',
+  description: 'Path to budgets.json'
+} satisfies Options;
```

Comment:
When you pass a value as default is will include this in the argvs if not it will pass it a empty var as undefiened. 

So if you want to handle weather it passed it a var or not, using this is complicated.

Not sure about the logic here. I do not know if this specific case requires it being undefined tho, but that is the motivation with not passing it a default.